prombench/manifests/cluster_gke.yaml

```diff
@@ -11,5 +11,5 @@ cluster:
       machinetype: n1-standard-4
       imagetype: COS
       disksizegb: 300
-      labels:
-        node-name: main-node
+    labels:
```

Comment:
unsure about this, what does this change do?

Reply:
![Screenshot from 2020-07-26 21-14-55](https://user-images.githubusercontent.com/42701709/88483388-1c436400-cf85-11ea-83ec-956480dc477c.png)

I am not sure why github is showing then unaligned. Can you just pull it and check it again?